app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -60,15 +65,19 @@ protected void onCreate(Bundle savedInstanceState) {
 
         fAuth = FirebaseAuth.getInstance();
         fStore = FirebaseFirestore.getInstance();
-        user = fAuth.getCurrentUser();
-        if (user != null) {
-            userId = user.getUid();
+
+        userManager = new FirestoreUserManager(fStore, FirestoreUserManager.USERS_COLLECTION, new UserSerializer());
```

Comment:
Do you really make use of this userManager here in EditProfile?

Reply:
Ok! Yes, we still have work to do here, you can take inspiration of RegisterActivity in my branch that you merge to check how to set values to Firebase Firestore 😉